src/entries/claim_update.rs

```diff
@@ -1,21 +1,23 @@
+use crate::entries::helper::{
+    generate_claim_key, generate_claim_value, generate_hold_key, generate_hold_value,
+    generate_withdrawal_key, generate_withdrawal_key_v1, generate_withdrawal_value,
+    generate_withdrawal_value_v1,
+};
+use crate::entries::smt::{generate_history_smt, save_smt_root_and_leaves};
 use crate::models::withdrawal::{get_withdrawal_cota_by_lock_hash, WithdrawDb};
 use crate::request::claim::ClaimUpdateReq;
-use crate::smt::common::{
-    generate_claim_key, generate_claim_value, generate_history_smt, generate_hold_key,
-    generate_hold_value, generate_withdrawal_key, generate_withdrawal_key_v1,
-    generate_withdrawal_value, generate_withdrawal_value_v1,
-};
+use crate::smt::db::cota_db::CotaRocksDB;
 use crate::utils::error::Error;
 use cota_smt::common::*;
 use cota_smt::molecule::prelude::*;
 use cota_smt::smt::{blake2b_256, H256};
 use cota_smt::transfer_update::ClaimUpdateCotaNFTEntriesBuilder;
 use log::error;
 
-pub fn generate_claim_update_smt(
+pub async fn generate_claim_update_smt(
     claim_update_req: ClaimUpdateReq,
 ) -> Result<(String, String), Error> {
```

Comment:
suggest to refactoring the return type of all `entries#generate_xxx_smt` fn, we should return `H256` / `ClaimCotaNFTEntries` directly, using string here is not easy to write unit test for these fn.